modules/core/src/v2/wallet.ts

```diff
@@ -1894,6 +1894,71 @@ export class Wallet {
     }).call(this).asCallback(callback);
   }
 
+  /**
+   * Sign a metamask transaction and send to wallet platform for pending approval.
+   *
+   * @param params
+   * @param callback
+   */
+  signAndSendMMITransaction(params: any/* TODO: add type */, callback?: NodeCallback<any>): Bluebird<any> {
```

Comment:
Can we convert this to async/await syntax inside this file or are there complications? Otherwise lets move to a new file